en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -12,7 +12,7 @@ the server to update templates, website logic, and perform upgrades or maintenan
 
 ## PHP
 
-* PHP >=7.4
+* PHP >=7.4, <8.2
```

Comment:
PHP 8.2 is officially _unsupported_ in CMS4. May as well make that explicitly clear.